_lessons/sources.html

```diff
@@ -80,7 +83,12 @@
         identifier: orders
 
 ```
+???
+Demo: Jump into dbt cloud to walk through this source.yml in the IDE.
+
+Note: Code in the project is different from on the slide. Slide contains schema and table identifiers - call this out as optional config that allow for more flexibility - giving you the chance to name a source something more appropriate than its given name in the db.
 
+Teaching Note: In case anyone asks 'What does the 'version: 2' in all the yml file code snippets do? Can/should I update yml code versions?'. Answer: "Once upon a time, the structure of these .yml files was very different (s/o to anyone who was using dbt back then!). Adding version: 2 allowed us to make this structure more extensible." taken from dbt docs https://docs.getdbt.com/faqs/why-version-2
```

Comment:
appreciation: definitely helpful for newer instructors - we almost always get asked this.